flocker/apiclient/_client.py

```diff
@@ -760,15 +801,23 @@ def _parse_configuration_container(self, container_dict):
         :return: ``Container`` instance.
         """
         return Container(
-            node_uuid=UUID(hex=container_dict[u"node_uuid"], version=4),
+            node_uuid=UUID(hex=container_dict[u"node_uuid"]),
```

Comment:
This change is not really related to this PR. But setting the version of a UUID only makes sense if we are minting a new UUID from some data. Overriding the version when creating a UUID instance from existing UUID data is asking for trouble.